services/response_code.proto

```diff
@@ -1116,4 +1116,19 @@ enum ResponseCodeEnum {
    * type actually has.
    */
   UNEXPECTED_TOKEN_DECIMALS = 283;
+
+  /**
+   * The proxy account id is invalid or does not exist.
+   */
+  INVALID_PROXY_ACCOUNT_ID = 284;
+
+  /**
+   * The transfer account id in CryptoDelete transaction is invalid or does not exist.
+   */
+  INVALID_TRANSFER_ACCOUNT_ID = 285;
+
+  /**
+   * The fee collector account id in TokenFeeScheduleUpdate is invalid or does not exist.
+   */
+  INVALID_FEE_COLLECTOR_ACCOUNT_ID = 286;
```

Comment:
We should update other places in the protobuf like INVALID_ALIAS_KEY, etc. that mention that alias only applies to crypto transfer.

Reply:
@steven-sheehy   `INVALID_ALIAS_KEY` , is still reserved only for CryptoTransfer that auto creates an account , if the `alias` provided for auto account creation fails to be parsed into `Key` .

Reply:
Can't all transaction types that take an `AccountID` somewhere in the transaction body now provide an alias key and thus fail if the key is invalid?

Doesn't the `alias` in the `TransactionRecord` now apply to any transaction type and need its documentation updated to reflect that?

Reply:
@steven-sheehy , Every transaction with an `AccountID` can use it in `alias` form. 
1. But the auto account creation only happens with `CryptoTransfer` to `alias`. During that transaction if the given `alias` is not a valid public key , the response is `INVALID_ALIAS_KEY` .
2. Once account is created and it is used in `alias` form in other transactions, its corresponding `accountNum` from the rebuilt data structure is looked up. If the `alias` is invalid in this scenario proper response codes like `INVALID_PAYER_ACCOUNT_ID` etc., are returned based on where the alias form is used. Because `alias` can be used in multiple places in single Transaction and `INVALID_ALIAS_KEY` doesn't provide enough info. 
 Added response codes in this PR are for the same. 
3. Afaik `alias` in child `TransactionRecord` is populated when auto account is created from `CryptoTransfer`. This doesn't change.

Reply:
As discussed, Every transaction with an AccountID can use it in `alias` form.

1. But the auto account creation only happens with `CryptoTransfer` to alias. During that transaction if the given `alias` is not a valid public key , the response is `INVALID_ALIAS_KEY` .
2. Once account is created and it is used in `alias` form in other transactions, its corresponding `accountNum` from the rebuilt data structure is looked up. If the `alias` given is invalid in this scenario proper response codes like `INVALID_PAYER_ACCOUNT_ID` etc., are returned based on where the alias form is used. Because alias can be used in multiple places in single Transaction and `INVALID_ALIAS_KEY` doesn’t provide enough info.
 Added response codes in this PR are for the same.
3. Afaik alias in `child TransactionRecord` is populated when auto account is created from `CryptoTransfer`. This doesn’t change.